src/api/app/components/accordion_reviews_component.html.haml

```diff
@@ -0,0 +1,23 @@
+.accordion#accordion-reviews-component
+  - if @pending_reviews_count.positive?
+    .accordion-item
+      %h2.accordion-header#heading-pending-reviews
+        = button_tag(class: 'accordion-button',
```

Comment:
I would start `collapsed` here. The pending reviews information is already available at the top of the right column. It is not so relevant for taking decision for the request.

Reply:
The whole point of this PR is to avoid having to scroll up in order to know who reviewed or hasn't yet, then you can take a fully informed decision.

Reply:
Okay, if there is a user-perspective reason for that, I'm fine with it.
I'd say in our (OBS) case the approach to reviews is slightly different from GitHub or other tools with reviews: at a first glance I'd be more interested in _who accepted/rejected it_ in order to merge it or fix it, and I wouldn't care about _who's still missing to the party_ , but for an OBS request probably that's the other way around it. Am I reading it right?

Reply:
Yes